internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -103,8 +111,28 @@ func (s *LoadBalancedWebService) Template() (string, error) {
 	return content.String(), nil
 }
 
+func (s *LoadBalancedWebService) loadBalancerTarget() (*string, *string) {
+	containerName := s.name
+	containerPort := strconv.FormatUint(uint64(aws.Uint16Value(s.manifest.Image.Port)), 10)
+	// Route load balancer traffic to main container by default.
+	targetContainer := aws.String(containerName)
+	targetPort := aws.String(containerPort)
+	mftTargetContainer := s.manifest.TargetContainer
+	if mftTargetContainer != nil {
+		sidecar, ok := s.manifest.Sidecars[*mftTargetContainer]
+		if ok {
+			targetContainer = mftTargetContainer
+			targetPort = sidecar.Port
+		} else {
+			log.Infof("Target container %s doesn't exist. Use main container to route traffics instead.\n", *s.manifest.TargetContainer)
+		}
```

Comment:
should this be an error? In general, I don't think we should log anything that's not `Debug` in packages other than `cli`. 
If as a customer I set a target container that doesn't exist, then erroring seems okay to me. What do you think?

Reply:
> In general, I don't think we should log anything that's not Debug in packages other than cli.

that's a good idea! Agree

Reply:
done!